src/condition-rule.ts

```diff
@@ -10,7 +10,7 @@ export class ConditionRule extends Rule {
 	assert: (this: Entity) => boolean;
 
 	// The message describing the state of the condition
-	message: string | ((this: Entity) => string);
+	message: string | ((this: Entity) => string | null | undefined);
```

Comment:
_info_: Allow `message` and `error` functions to return `null` or `undefined` to avoid TS errors, and the rules should handle null/undefined anyway.